src/components/interview.ts

```diff
@@ -9,8 +9,12 @@ export const availableDomains: { [key: string]: string } = {
   backend: 'Backend',
   design: 'Design',
   pm: 'PM',
-  datasci: 'Datasci',
-  infra: 'Infra'
+  datasci: 'DataSci',
+  infra: 'Infra',
```

Comment:
Stupid nit: Can we make this alphabetical? So it's easier to add/remove from and probably easier for the user to find their domain.